RequestService/RequestService.AzureFunction/Startup.cs

```diff
@@ -95,6 +98,9 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.TryAdd(ServiceDescriptor.Singleton(typeof(ILogger<>), typeof(Logger<>)));
             builder.Services.TryAdd(ServiceDescriptor.Singleton(typeof(ILoggerWrapper<>), typeof(LoggerWrapper<>)));
 
+            builder.Services.AddMemCache();
```

Comment:
Comparing with other services, we may also need the following before AddMemCache():
`services.AddSingleton<IPollyMemoryCacheProvider, PollyMemoryCacheProvider>();`